.github/workflows/post-merge-ci.yml

```
@@ -122,23 +155,271 @@ jobs:
+    runs-on: prod-default-small-v2
+    needs: [deploy-operator, vllm-pipeline]
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        profile: [agg, agg_router, disagg, disagg_router]
+    name: deploy-test-vllm (${{ matrix.profile }})
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run Dynamo Deploy Test
+        uses: ./.github/actions/dynamo-deploy-test
```

actionlint (1.7.11)

[error] missing input "registry" which is required by action "Dynamo Graph Deployment Test" defined at "./.github/actions/dynamo-deploy-test". all required inputs are "framework", "image", "kubeconfig_base64", "namespace", "profile", "registry"

(action)